playbooks/manager/copy-ceph-keys.yml

```diff
@@ -39,5 +39,10 @@
         INTERACTIVE: false
 
     - name: Copy ceph keys to the configuration repository
-      ansible.builtin.command: "docker cp {{ ceph_ansible_container_name }}:/share/{{ ceph_cluster_fsid }}/etc/ceph/{{ item.src }} {{ item.dest }}"  # noqa no-changed-when
+      ansible.builtin.command: "docker cp {{ ceph_ansible_container_name }}:/share/{{ ceph_cluster_fsid }}/etc/ceph/{{ item.src }} {{ item.dest }}"
+      changed_when: false
```

Comment:
I would prefer to set `changed_when` to true, as "docker cp" can perform changes on the system.